tutorials-v5/time-evolution/023_v5_paper-mesolve.md

```diff
@@ -0,0 +1,565 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# QuTiPv5 Paper Example: `sesolve` and `mesolve` and the new solver class
+
+Authors: Maximilian Meyer-Mölleringhof (m.meyermoelleringhof@gmail.com), Neill Lambert (nwlambert@gmail.com)
+
+In QuTiP, the `Qobj` and `QobjEvo` classes form the very heart of almost all calculations that can be performed.
+With these classes, open quantum systems with various interactions and structures can be simulated by using the wide variety of solvers provided.
+Most of the time, these solvers are given an initial state, a Hamiltonian and an environment that is often described using rates or coupling strengths.
+QuTiP then uses numerical integration to determine the time evolution of the system.
+
+QuTiP v5 introduces a unified interface for interacting with these solvers.
+This new interface is class-based, allowing users to instantiate a solver object for a specific problem.
+This can be useful when the same Hamiltonian data is reused with different initial conditions, time steps or other options.
+A noticable speed-up can thus be achieved if solvers are reused multiple times.
+
+Upon instantiation, one first supplies only the Hamiltonian and the collapse operators (e.g., collapse operators for a Lindbladian master equation).
+Initial conditions, time steps, etc. are passed to the `Solver.run()` method which then performs the simulation.
+
+In this notebook we will consider several examples illustrating the usage of the new solver classes.
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+from qutip import (QobjEvo, SESolver, UnderDampedEnvironment,
+                   about, basis, brmesolve, fidelity, mesolve,
+                   qeye, sigmam, sigmax, sigmaz, spost, spre, sprepost)
+
+from qutip.solver.heom import HEOMSolver
+
+%matplotlib inline
+```
+
+## Part 0: Introduction to the New Solver Class
+
+In our first example, we want to look at two interacting qubits that are (for now) decoupled from an environment.
+Such a system is described by the Hamiltonian
+
+$H = \dfrac{\epsilon_1}{2} \sigma_z^{(1)} + \dfrac{\epsilon_2}{2} \sigma_z^{(2)} + g \sigma_{x}^{(1)} \sigma_{x}^{(2)}$.
```

Comment:
```suggestion
$$ H = \frac{\epsilon_1}{2} \sigma_z^{(1)} + \frac{\epsilon_2}{2} \sigma_z^{(2)} + g \sigma_{x}^{(1)} \sigma_{x}^{(2)} . $$
```
I noticed you avoid `$$ ... $$`, I was wondering what the reason is

Reply:
No particular reason, the other notebooks where not super consistent either so I just chose one. I guess both are fine as long as they are consistent within one notebook

Reply:
Ah, I see. Okay, up to you too choose!